brendenwest/streetends
# User location

closes #60  User Story: "As a user, I want to see my location so that I can view the closest street end on the map"

AC:
display the current location on the map with different color pin
location permissions

SUMMARY:
- Add Android Permissions
- Shows user location on the map if the user granted access
- Geolocation data for user's lat and longitude for future use to implement finding closest street ends and zoom in feature.
- Button with GPS icon to call location data

TEST CASE: 
1) Run Street ends app
2) Open map via bottom navigation, second from left there is a map icon.
2) Turn on location settings when prompted. This only happens the first time running the app.
3) Blue dot appears where your current location is set to.
**If you need to set your current location: 
    a. Open emulator settings
    b. Go to "location" - first in the top left navigation
    c. Click your destination (around Seattle preferably to see markers on street end map)
    d. Click set location bottom right to set new current location
    e. go back to the app
4) Click the blue GPS button at the bottom of Map Screen above the navigation bar
5) Check the console log for the latitude and longitude of your set location
![UserLocation](https://user-images.githubusercontent.com/49288075/140314754-5767fa8c-e429-4aba-9733-351687f1e60e.PNG)
